web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -30,6 +30,18 @@ function ExchangeArrow({
   const setIsMoving = useSetAtom(mapMovingAtom);
   const headerHeight = useHeaderHeight();
 
+  useEffect(() => {
+    const cancelWheel = (event: Event) => event.preventDefault();
+    const exchangeLayer = document.querySelector('#exchange-layer');
+    if (!exchangeLayer) {
+      return;
+    }
+    exchangeLayer.addEventListener('wheel', cancelWheel, {
+      passive: true,
+    });
+    return () => exchangeLayer.removeEventListener('wheel', cancelWheel);
+  }, []);
```

Comment:
Even more use effect hooks 🙈 

I'll test this after lunch!

Reply:
this one was already there, I had removed it tho! I also didn't have a mousewheel to test 😆